test/integration/admin_views_all_orders_test.rb

```diff
@@ -0,0 +1,22 @@
+require "test_helper"
+
+class AdminViewsAllOrdersTest < ActionDispatch::IntegrationTest
```

Comment:
Can we change the filename & the test name to "Admin Views An Individual Order" so it matches the title/card for issue #75?